lib/abgabe/abgabe_client_lib/test/bloc_test.dart

```diff
@@ -834,9 +834,6 @@ void main() {
         });
       });
     },
-    timeout: Timeout(
-      Duration(seconds: 5),
-    ),
```

Comment:
I don't know for sure anymore.    
My guess is that because it's a bloc and thus  it uses async events that instead of tests just failing they will wait indefinitely for an event that has yet to come (that is expected). So instead of letting the tests/ci-job run for until the default timeout is hit, we use a short timeout instead (I think the default is maybe like 10 mins or sth?).   

I think this is actually useful, we should keep it imo.

Reply:
Default should be 30 seconds:

> By default, tests will time out after 30 seconds of inactivity. The timeout applies to deadlocks or cases where the test stops making progress, it does not ensure that an overall test case or test suite completes within any set time.

https://github.com/dart-lang/test/tree/master/pkgs/test#timeouts

Reply:
For unit tests (`test()`) it's 30 seconds, for widget tests (`testWidgets()`) it's 10 minutes (uses `TestWidgetsFlutterBinding.defaultTestTimeout` which is set to 10 minutes, see: https://api.flutter.dev/flutter/flutter_test/TestWidgetsFlutterBinding/defaultTestTimeout.html).

Demo: https://zapp.run/edit/flutter-zn8a0669n8b0?entry=test/widget_test.dart&file=test/widget_test.dart

In this file we are only using unit tests (`test`).

Reply:
Hm, ok